src/lightning_app/cli/lightning_cli_add.py

```diff
@@ -0,0 +1,38 @@
+import os
+from pathlib import Path
+from typing import Union
+
+import click
+from lightning_cloud.openapi.rest import ApiException
+
+from lightning_app.cli.cmd_ssh_keys import SSHKeyManager
+
+
+@click.group("add")
+def cli_add() -> None:
+    """Add Lightning AI self-managed resources (ssh-keys, etc…)"""
+    pass
+
+
+@cli_add.command("ssh-key")
+@click.option("--name", "key_name", default=None, help="name of ssh key")
+@click.option("--comment", "comment", default="", help="comment detailing your SSH key")
+@click.option(
+    "--public-key",
+    "public_key",
+    help="public key or path to public key file",
+    required=True,
+)
+def add_ssh_key(key_name: str, comment: str, public_key: Union[str, "os.PathLike[str]"] = None) -> None:
```

Comment:
```suggestion
def add_ssh_key(key_name: str, comment: str, public_key: Union[str, "os.PathLike[str]"]) -> None:
```

Shouldn't public_key be the first argument, since it is required?
And key_name should become Optional[str]

Reply:
yup, done.